lib/pages/chat/chat_event_list.dart

```diff
@@ -151,4 +145,24 @@ class ChatEventList extends StatelessWidget {
       ),
     );
   }
+
+  Widget _chatEmptyBuilder() {
+    if (controller.room?.isDirectChat ?? true) {
+      return DirectDraftChatView(
+        onTap: () => controller.inputFocus.requestFocus(),
+      );
+    } else {
+      return _groupChatEmptyBuilder();
+    }
+  }
+
+  Widget _groupChatEmptyBuilder() {
+    if (controller.timeline!.events.isNotEmpty) {
```

Comment:
should check timeline is != null also

Reply:
PLEASE READ CAREFULLY TO HELP ME THANKS

<img width="531" alt="Screenshot 2023-08-16 at 13 27 23" src="https://github.com/linagora/twake-on-matrix/assets/99852347/cf4ba096-7711-4ebd-b02f-3f60c13206f5">